cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,57 @@
 /// <reference types='cypress' />
+import { generateWorker } from '../support/generate';
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should check pagination', () => {
+    cy.get('.-pageInfo').should('be.visible').and('exist');
+    cy.get('[aria-label="rows per page"]').should('be.visible').and('exist');
+  });
+
+  it('should add new user', () => {
+    const { email, firstName, lastName, age, salary, department } =
+   generateWorker();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(firstName);
+    cy.get('#lastName').type(lastName);
+    cy.get('#userEmail').type(email);
+    cy.get('#age').type(age.toString());
+    cy.get('#salary').type(salary.toString());
+    cy.get('#department').type(department);
+    cy.get('#submit').click();
+    cy.get('.rt-table').should('contain', firstName, lastName, email, age,
+      salary, department);
+  });
+
+  it('should verify that the user deleted correctly', () => {
+    cy.get('#delete-record-2').click();
+    cy.get('.rt-table').should('not.contain', '#delete-record-2');
+  });
+
+  it.only('should verify that all the users deleted correctly', () => {
+    cy.get('[title="Delete"').click({ multiple: true });
+    cy.get('.rt-table').should('not.contain', '#delete-record-2');
```

Comment:
You're checking that the table does not contain '#delete-record-2' after deleting all users. However, '#delete-record-2' is an id, not a text content. It would be better to check that the table does not contain any rows or cells.